packages/video_player/video_player_avfoundation/darwin/video_player_avfoundation/Sources/video_player_avfoundation/FVPVideoPlayer.m

```diff
@@ -370,52 +362,15 @@ - (void)sendFailedToLoadVideoEvent {
 }
 
 - (void)reportInitializedIfReadyToPlay {
```

Comment:
This should just be `reportInitialized` now, as there is no more "if" aspect to its behavior.